eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,1271 @@
+        "prefLabel": {
+          "en": "Course completed"
+        },
+        "definition": {
+          "en": "Learner has completed the course."
+        },
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.name",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.result.completion",
+            "any": [true]
+          },
+          {
+            "location": "$.verb.id",
+            "any": ["http://adlnet.gov/expapi/verbs/completed"]
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/courseInitialized",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "prefLabel": {
+          "en": "Course initialized"
+        },
+        "definition": {
+          "en": "Learner has initialized the course."
+        },
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.name",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.verb.id",
+            "any": ["http://adlnet.gov/expapi/verbs/initialized"]
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/courseLaunched",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "prefLabel": {
+          "en": "Course launched"
+        },
+        "definition": {
+          "en": "Learner has launched the course."
+        },
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.verb.id",
+            "any": ["http://adlnet.gov/expapi/verbs/launched"]
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.name",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.type",
+            "any": ["http://adlnet.gov/expapi/activities/course"]
```

Comment:
this one should probably be the determining property objectActivityType

Reply:
A `scopeNote` was added to every template that uses this property specify this.